lib/ci-config-stack.ts

```diff
@@ -28,12 +28,24 @@ export class CIConfigStack extends Stack {
     constructor(scope: Construct, id: string, props?: StackProps) {
       super(scope, id, props);
 
-      const arnSecret = new Secret(this, 'certificateArn', {});
-      const certContentsSecret = new Secret(this, 'certContents', {});
-      const certChainSecret = new Secret(this, 'certChain', {});
-      const privateKeySecret = new Secret(this, 'privateKey', {});
-      const redirectUrlSecret = new Secret(this, 'redirectUrl', {});
-      const OIDCConfigValuesSecret = new Secret(this, 'OIDCConfigValues', {});
+      const arnSecret = new Secret(this, 'certificateArn', {
+        description: 'Certificate ARN retrieved after uploading certificate to IAM server',
+      });
+      const certContentsSecret = new Secret(this, 'certContents', {
+        description: 'Contents of public key of the SSL certificate',
+      });
+      const certChainSecret = new Secret(this, 'certChain', {
+        description: 'Contents of the SSL certificate chain',
+      });
+      const privateKeySecret = new Secret(this, 'privateKey', {
+        description: 'Contents of private key of the SSL certificate',
+      });
+      const redirectUrlSecret = new Secret(this, 'redirectUrl', {
+        description: 'Redirect url for Jenkins',
```

Comment:
I think it's a good idea to add a default value as well for this secret as we discussed offline.

Reply:
I did not find a direct way of doing that. I see this feature https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-secretsmanager.SecretStringValueBeta1.html which was implemented `v1.138.0` and later